[8.0][mrp_partial_production] Added new module to partial production

josemoralesp opened the pull request:
In some case is needed start a production with partial reservation because you
can deliver your order in different parts. This module allow process a
manufacturing order with partial reservation, producing multiple times until
the order is completed.

The module add a new compute field to show the quantity available to produce
considering the number of the reservation in the products to consume. This
field is considered in all process and you cannot produce more than this field
indicates.

This module don't allow the creation of negative quant, this mean you cannot
force the production.

Commit:
[IMP] Added new module to allow creaate partial productions
[IMP] Added new field to compute the quantity available to produce considering the reservation in the moves

Commit:
[IMP] Change the logic to determinate if a PO is ready to be produced. If we have at least reserve enough to product one peace we can start the production

Commit:
[IMP] Added new method to avoid an direct acoint over action_produce method and produce more than available

Commit:
[IMP] Set the correct inheritance

Commit:
[IMP] Shown the button to reserve when the production is started

Commit:
[IMP] Considering the quantity  produced to compute the available quantity to produce

Commit:
[IMP] Added comments in methods

Commit:
[IMP] Added demo data needed for the test process

Commit:
[IMP] Added sale to validate all process

Commit:
[IMP] Added method to run procurements and create the manufacturing order

Commit:
[ADD] Added inventory to load quants for the test products

Commit:
[IMP] Added new method to avoid create and inheritance over a function field
[IMP] Change the field to stored to compute only whene the resevation are changed

Commit:
[ADD] Added new test to validaty to process modified
[ADD] Added translations files

nhomar commented on mrp_partial_production/__init__.py:
```diff
@@ -0,0 +1,30 @@
+##############################################################################
+from . import models
+from . import wizard
+# from . import tests
```
tests is commented, that's ok?

josemoralesp replied:
Yes, the test files into the folder tests are executed automatically[1]. It is unnecessary  import the folder.

https://drive.google.com/a/vauxoo.com/file/d/0B2kzKLGF6ZvLalF0Q21CNjJ5bGc/view?usp=drivesdk

Commit:
[IMP] Removed the way to determinate the quantity produced because there is a method in mrp to get that value
[IMP] Removed the store in the field to avoid prodblem at the momento to compute the reamaining value with orders with all lines reserved and partial production

josemoralesp commented:
@nhomar Can you merge this please.

@JulioSerna  and Juan Aguilar tested the functionality and they approve the module.
